docker/configurator/variables.js

```diff
@@ -0,0 +1,113 @@
+const standardVariables = {
```

Comment:
If there's a reason to reduce which of these are being added, please let me know.